### DISCUSSION TITLE
شورای امنیت سازمان ملل

### DISCUSSION THREAD

**Discussion by AnonymousMaximus2** [WeAreMahsaAmini/Core#271] [Ideas]
شاید این نوشته یک ایده نباشد بلکه کمک برای هدایت و جهت دهی تمامی ایده ها باشد

1- یگان ویژه جمهوری اسلامی برای موارد بسیار خطرناک تر آموزش دیده

2- ارتش ایران در اصل همان افراد آموزش دیده سپاهیان و اطلاعاتی ها و نیروی انتظامی است که ظاهرش فرق دارد 

3- ایران طبق قراردادهای بین المللی با روسیه و چین یک نیروی نظامی بین المللی متحد مقابل ارتش ناتو اروپا و آمریکا ایجاد کرده است.

4- حکومت ایران از کشتن چند ده نفر سهله از کشتن چند میلیون نفر نیز باکی ندارد.

پس چاره چیست ؟

 اگر خوب به اخبار و رسانه ها دقت کنید سازمانهای حقوق بشری و سازمان ملل و دولت های اروپا و آمریکا  حرفها و کلمات شان دقیق و سنجیده و حساب شده است ولی کشورهای روسیه و چین و طرفداران آنها سکوت کرده اند.

تعدادی از سازمانهای حقوقی بشری و شخصیتهای مهم بین المللی سعی دارند کشتارهای ایران و کشته شدن با گلوله را جنایت بین المللی معرفی کنند.

اما چرا؟

در اصل اینجا یک جنگ رسانه ای ایجاد شده است تا با گردآوری اسناد و مدارک تمامی جنایتها و کشانیدن این جنایتها به سازمان ملل

در نهایت این پرونده ها در شورای امنیت سازمان ملل مطرح شود.

زمانی که شورای امنیت سازمان ملل تایید کند که در ایران جنایت جنگی علیه مردم ایران انجام شده است ، دولت ایران را محکوم خواهد کرد.

حال محکومیت ایران چه فایده ای دارد؟

اگر به فرایند حمله به صدام و لیبی قذافی و سایر کشورهای دنیا نگاه کنید، معمولا در چنین مواردی که مردم ضعیف مقابل دولت خود می ایستند و دولت دست به اقدامات ضد حقوق بشری میزند، 

1- در ابتدا سازمانهای حقوق بشری فریاد میزنند 

2- صدایشان به سازمان ملل میرسد.

3- موضوع در شورای امنیت سازمان ملل تصویب میشود که باید اقدامی صورت بگیرد

4- دادگاههای بین المللی  حکم صادر میکنند،

5- اگر 6 کشور عضو وتو کننده سازمان ملل وتو نکنند، در این صورت ارتشهای جهان با هم متحد شده و به آن کشور حمله کرده و حکومت را سرنگون و حکومتی مردمی با رای گیری و رفراندوم تشکیل میدهند.

در اینجا به نظر میرسد تمامی صحبتها و حرفهای سازمانهای حقوق بشری بین المللی و رفتار و دیپلماسی دولتهای اروپا و آمریکا با هم همسو برای کشانیدن موضوع شورشهای ایران به سازمان ملل و در نهایت تصویب در شورای امنیت سازمان ملل ااست.

اگر در اینجا روسیه و چین وتو نکنند و یا به نوعی سکوت کنند ارتشهای اروپا و آمریکا متحد شده و به ایران حمله و حکومت را تحویل مردم خواهند داد.

همان اتفاقی که در کشورهایی مانند عراق و لیبی و سایر کشورها افتاد.

تنها مانع این حرکت وتو چین و روسیه در سازمان ملل از یک طرف

و دومی پراکنده کاری مردم و طرفداران مردم ایران است.

اگر مردم و طرفداران مردم ایران با هم متحد شده و فعالیتهایشان را همسو کنند موضوع در سازمان ملل با قوت و قدرت بیشتری میتواند مطرح شده و منجر به سکوت و یا کناره گیری چین و روسیه از جلوگیری از حمله شود.

ولی اینکه مردم بتوانند با دست خالی بر چنین حکومتی پیروز بشوند، فقط یک معجزه میتواند باشد.

**Comment by azinbr** [comment]
ابتدا میخوام بگم هرگز قدرت مطلق خداوند رو فراموش نکنیم 

بدن ما انسانها و عملکردش خودش بزرگترین معجزه ی هستیه ما خوئ اون معجزه هستیم

فراموش نکنیم این حکومت از جنبه های زیادی تحت فشار هست اقتصادی ، اجتماعی ، بین المللی  و... از طرفی بزرگترین قدرت هر حکومت رضایت مردمه پس

اگه یه فضایی فراهم کنیم که میزان جراحت هرکسی که آسیب دیده یا فیلمهای رویدادها مستقیم به دست سازمان عفو بین الملل برسه و یه رفراندوم برای مردم بزاریم که مشخص بشه خواسته ی اکثریت بر بودن این حکومته یا رفتنشون خیلی میتونه از هزینه ی جانی مردم کم کنه چون روند  سرعت میگیره

از طرفی این خیزش نیاز به هدایت داره اگه بتونیم فضای امنی برای گفتگو با معترضین حاضر در خیابون ها داشته باشیم  خیلی کمک کننده ست

اگه هر کسی بر اساس شهر محل سکونتش بتونه قبل از رفتن به خیابون بدونه برنامه چیه و بدونه فضا امن هست خیلی میتونیم کمک کنیم و از دسیسه ها جلوگیری بشه و دستگیریها رو کم کنیم
یادمون نره تعداد ما از اونها خیلی بیشتره

**Comment by AnonymousMaximus2** [comment]
1- حکومت آمارها را داره و طبق برخی آمارهایی که گاها به بیرون درز میکنه  بالای 80 درصد مردم از حکومت ناراضی هستند یعنی 65 میلیون نفر، و حکومت با داشتن 2 میلیون نفر نیروی نظامی که عملا چند صد هزار نفرشون فعال هستند و چند میلیون نفر افراد بوق طرفدار حکومت که اصلا توی باغ نیستند و توی عالم هپروت سیر میکنند، و دین تحریف شده توی مغزشان فرو شده ، بقیه افراد طرفدار حکومت نیستند.

2- در انقلاب 57 هم تعداد کمی شورش کردند و انقلاب کردند و بقیه سکوت کرده بودند، اگر به فیلمهای انقلاب سال 57 نگاه کنید کل جمعیت ایران آن زمان 35 میلیون نفر بود ولی کل راهیمائی کننده ها به چند صد هزار نفر نمیرسید، یعنی حدود 90 درصد مردم سکوت کرده بودند.

3- گفتن اینکه میزان جراحت هر کسی که آسیب دیده خیلی راحته ، ولی اگر مثل من یا افرادی که جراحت عمیق روحی دیدین و درونتان و وجودتان پر از خشم و انتقام و کینه و نفرت از طاغوتی به نام جمهوری اسلامی است، و شبانه روز برای درمان بیماری افسردگی تان باید این جرثومه های فساد را که در لباس انسانیت و دین و اسلام میخورند و میخوابند و دنیا را به بازی گرفته اند را میدیدید اون وقت میفهمیدین که گاهی بعضی چیزها بخشودنی نیستند.

اما دلیل نمیشه وقتی جراحت عمیق داری بری وسط میدان و خودت را به کشتن بدی، من مطمئنم 100 برابر افرادی که به خیابانها ریخته اند، افرادی در جامعه ایران هستند که خشمشان را خورده اند و به قول حضرت علی ع مانند استخوان در گلو منتظر فرصتی برای گرفتن انتقام عزیزان از دست رفته شان هستند. و البته این عده که سکوت کرده اند تیز هوش تر ، و تیز بین تر و عاقلانه تر رفتار میکنند و منتظر فرصت هستند، 

افرادی که به خیابان ریخته اند البته قابل تحسین هستند، ولی ضربه نهائی و اصلی از سوی آنها که ظلم دیده و سکوت کرده اند وارد خواهد شد.

اگر به تخریب یک ساختمان دقت کنید، شیشه و پنجره و در ساختمان وقتی میشکند سر و صدا میکند ولی ساختمان سرجایش است.

اما وقتی یکی از ستونهای ساختمان پوسیده و یا بشکند، هیچ صدائی ندارد و تنها صدایش فرو ریختن ساختمان است.

در اینجا نیز آنها که سکوت کرده اند تعدادی از آنها قدرت سکوت شان یک نفرشان به اندازه شعار یک میلیون نفر است.
